SPECS/uclibc/uclibc.spec

```diff
@@ -0,0 +1,198 @@
+%global uclibc_name uClibc
```

Comment:
Nit: seems unnecessary. The source URL is case-insensitive, so we can just directly use this for the `%setup` macro.